DependencyInjection/Compiler/TweakCompilerPass.php

```diff
@@ -31,12 +31,31 @@ public function process(ContainerBuilder $container)
 
         $parameters = $container->getParameter('sonata_block.blocks');
 
-        foreach ($container->findTaggedServiceIds('sonata.block') as $id => $attributes) {
+        foreach ($container->findTaggedServiceIds('sonata.block') as $id => $tags) {
```

Comment:
`$tags` is not used, you could write `array_keys($container->findTaggedServiceIds('sonata.block'))` instead, might be less confusing